docs/data/datasets/datamanagement.md

```diff
@@ -96,7 +96,7 @@ Budgeting and costing for your project is often dependent upon institutional res
     - Will you need additional paid help for organising and documenting data; are your data files, sheets etc. stored in uniform format and clearly named and well organised and understandable? Higher costs will apply if organising data has been neglected during the project. 
     - When you archive and open data, will there be costs involved?
 
-Read more about [CSC free-of-charge use cases and pricing](https://research.csc.fi/pricing).
+Read more about [CSC free-of-charge use cases and pricing](https://research.csc.fi/pricing-and-resources).
```

Comment:
Looks like there is a more specific page for "free-of-charge use cases"

```suggestion
Read more about [CSC free-of-charge use cases and pricing](https://research.csc.fi/free-of-charge-use-cases).
```

Reply:
> Looks like there is a more specific page for "free-of-charge use cases"

...but I now see that the link text has "and pricing" on it, so I'll leave it as it is.